### DISCUSSION TITLE
Question: When scrolling, values for hint position are not changing

### DISCUSSION THREAD

**Discussion by lukyncze** [hakimio/ngx-ui-tour#97] [Q&A]
Hi, I would like to ask if you don't know why the values for hint position are not changing while scrolling? Specifically **top** and **left** props in the picture. Does anyone have the same problem or could it be because of something else in my app?

Thanks in advance!

![image](https://user-images.githubusercontent.com/36028726/208860825-e58a6426-345d-4de0-901a-33dd6c8720b5.png)

**Comment by hakimio** [comment]
Don't know. The positioning is handled by [`mat-menu`](https://material.angular.io/components/menu/overview) component and this library can only influence it with `disablePageScrolling` step config which is equivalent to `block` [MAT_MENU_SCROLL_STRATEGY](https://material.angular.io/components/menu/api#MAT_MENU_SCROLL_STRATEGY).

**Reply by lukyncze** [reply]
@hakimio thanks for your input. Unfortunately this didn't help me :(

**Reply by hakimio** [reply]
What are you trying to achieve and what's not working for you? Can you create a reproduction of your issue on StackBlitz or GitHub?

**Reply by lukyncze** [reply]
I tried using your library for rendering tour anchors (hints). This problem probably is on my side, because some service in the app is messing up with styles.

- When I am scrolling vertically hint is stucked on its starting position - absolutely positioned in site view (inline styles of top and left don't change).
- On the other hand when I scrolled the site horizontally, I realized that hint is working as expected (it is not just absolutely positioned in site view, but it is positioned on the element where it should be - inline styles are changing as it should).

I can't make a recording and reproduction won't help with this because it's probably something in the app....

**Reply by hakimio** [reply]
Maybe an issue with some of the polyfills you are using?
